handout/architect/service-outline.md

```diff
@@ -0,0 +1,49 @@
+# Services
+
+- Prelude (before giving you ivory tower mandates, let's look at why we might want services)
+
+- Code:
+  - Component with http calls in methods  
+    (~8 lines w/ ellipses)
+  - Point out (bullets or a sentence): Maintenance; reuse with different endpoints; caching data; complexity management
+
+- Talk about Refactoring by pulling self-contained processes into their own functions with arguments
+
+- Code:
+  - function that represents previous call, with data
+    (~3 lines)
+  - method which calls the function, passing data
+    (~5 lines w/ ellipses)
+  - Point out: testable; configurable; maintainable; still not reusable
+
+- Segue and introduce Services (tagged subsection)
```

Comment:
It was more a note to self - I had a prelude of what the pain-points were; this is where you would roll the title screen and production credits.